### DISCUSSION TITLE
Best place for a new contributor to get started?

### DISCUSSION THREAD

**Discussion by michael-hart** [ros2-rust/ros2_rust#363] [Q&A]
Hi folks! I love the project and want to start helping out. I'm looking for something small to get going with and wanted to ask if there are any suggestions.

One idea I had was to try and implement ROS logging instead of println, so everything goes through the ROS logs. Let me know what you think!

**Comment by jhdcs** [comment]
Hello and welcome!

If you wish to tackle ROS logging, that would be fantastic! That is certainly an area that would be quite useful, and I don't think we've really focused on it just yet... Feel free to create an issue for it (if it hasn't already been made) and have at it! And don't be afraid to ask questions about the project, whether in the issue or on our Matrix chat. If you don't have access to the Matrix chat, just let me know and I can send you a link.

Once again, welcome! Very glad to have you with us!
